### ISSUE TITLE
.pug file change does not trigger page reload

### ISSUE THREAD

**Issue by dum3ng** [parcel-bundler/parcel#2090]
# 🐛 bug report
change a .pug file should trigger page reload.

## 🎛 Configuration (.babelrc, package.json, cli command)

no configuration.

## 🤔 Expected Behavior
1, use `parcel index.pug`
2, edit index.pug and save.
The browser should automatically reload the page.

## 😯 Current Behavior
When the entry point is *html* type, the page reload works. But that does not work
for a *pug* file type.

## 💁 Possible Solution

For now I use 'parcel watch index.pug' and use another watch server, but I think parcel should
handle this natively.

## 💻 Code Sample
index.pug
```pug
<!DOCTYPE html>
html(lang="en")
  head
    meta(charset="UTF-8")
    meta(name="viewport", content="width=device-width, initial-scale=1.0")
    meta(http-equiv="X-UA-Compatible", content="ie=edge")
    title Document
  body
    h2 should update in browser after edit me 
    script(src="./ts/play.ts")
```
./ts/play.ts
```ts
console.log('parcel with pug');
```
## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 1.10.1
| Node             | 10.11.0
| Yarn         | 1.10.1
| Operating System | macos mojave 10.14

**Comment by Ty3uK** [comment]
Will be fixed in #2091 when PR is accepted

**Comment by Ty3uK** [comment]
@dum3ng PR is merged, check it out!
